content/en/opentelemetry/guide/switch_from_processor_to_connector.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Switch from Datadog Processor to Datadog Connector for OpenTelemetry APM Metrics
+kind: guide
+is_beta: true
+further_reading:
+- link: "/opentelemetry/guide/ingestion_sampling_with_opentelemetry/"
+  tag: "Documentation"
+  text: "Ingestion Sampling with OpenTelemetry"
+- link: "https://opentelemetry.io/docs/collector/build-connector/"
+  tag: "OpenTelemetry"
+  text: "Connectors in OpenTelemetry"
+---
+
+{{< callout url="" header="false" btn_hidden="true" >}}
+  Datadog connector for the OpenTelemetry Collector is in beta. 
+{{< /callout >}} 
+
+## Overview
+
+If you use the soon-to-be deprecated [Datadog Processor][1] in your OpenTelemetry Collector pipeline for APM metrics, switch to use the [Datadog Connector][2] instead. 
+
+The Datadog Connector for the OpenTelemetry Collector allows APM metrics to be calculated on 100% of the trace data, even when sampling is applied. It also complies with the OpenTelemetry standard better than the Datadog Processor does.
+
+## Switch your Agent OpenTelemetry Collector configuration
+
+To switch to using the Datadog Connector, change the following Agent configurations:
+
+1. Add the `connectors` section:
+{{< highlight yaml "hl_lines=7-14" >}}
+processors:
+   probabilistic_sampler:
+     sampling_percentage: 20
+     # Add the "datadog" processor definition
+  datadog:
+
+# Add this section
+ connectors:
+  # Add the Datadog Connector definition and further configurations
+  datadog:
+     exporters:
+        datadog:
+        api:
+        key: ${env:DD_API_KEY}
+{{< /highlight >}}
+
+3. Duplicate the `service.pipelines.trace` configuration for the sampled trace pipeline.
+4. Add the Datadog Connector to the duplicated pipeline configuration that isn't sampled:
+     {{< highlight yaml "hl_lines=3-6" >}}
+     service:
+       pipelines:
+         traces:
+           receivers: [otlp]
+           processors: [batch]
+           exporters: [datadog/connector]
+         traces/2: # This duplicated pipeline uses sampling
+           receivers: [otlp]
+           processors: [batch, probabilistic_sampler]
+           exporters: [datadog]
+     {{< /highlight >}}
+5. Add the Connector configuration to `metrics` section:
+   ```yaml
+       metrics:
+         receivers: [datadog/connector]
+         processors: [batch]
+         exporters: [datadog]
+   ```
+
+## Example
+
+Here's a full example with all of the new configuration:
+
+```yaml
+processors:
+  probabilistic_sampler:
+    sampling_percentage: 20
+
+ connectors:
+  # Add the "datadog" connector definition and further configurations
+   datadog:
```

Comment:
It needs to be called datadog/connector since just saying datadog would conflict with the other exporter datadog component